docs/test_env_setup.rst

```diff
@@ -21,34 +21,18 @@ repository`_. You need to deploy test machines using playbooks from there.
 Ansible Roles for Tendrl Setup
 ==============================
 
-Right now, we maintain ansible roles and playbook which automates production
+Right now, we maintain ansible roles and playbooks which automate production
 like installation of all Tendrl components. `Tendrl project wide
 documentation`_ is followed, but most details are based on documentation or
 README files of each component.
```

Comment:
This paragraph should be removed, it's no longer true that we care about documentation linked there or about readme files of particular components.